app/views/content/blog/product-engineer-turned-science-teacher.md

```diff
@@ -0,0 +1,61 @@
+---
+title: Product engineer turned science teacher
+date: "2022-07-18"
+images:
+  chris_wallis:
+    path: "media/images/content/blog/thumbnails/chris-wallis.jpg"
+    thumbnail_path: "media/images/content/blog/thumbnails/chris-wallis.jpg"
+author: Chris Wallis
+description: |-
+  Chris Wallis talks about his career change from a product engineer to a science teacher, and how he uses his engineering skills and experience in the classroom.
+keywords:
+  - teacher training advisers
+  - becoming a teacher
+  - applications
+  - teacher training
+  - career changer
+  - science teacher
+tags:
+  - becoming a teacher
+  - career change to teaching
+---
+
+## What inspired you to become a teacher?
+
+I was very good at maths at school but loved physics, so a career in engineering seemed the obvious choice.
+
+I thoroughly enjoyed my first role in industry as a production engineer, but a promotion took me away from the practical engineering and I no longer enjoyed what I was doing.
+
+I tried many things until I realised that working with young people and helping them develop skills truly inspired me. So I applied for a PGCE at a University.
```

Comment:
Hi @EllieNodder - when it's not a named university, should it be lower case u?

Reply:
oops - yes! I think it said the name which I took out and forgot to take out the capital!